tardis/io/logger/logger.py

```diff
@@ -136,6 +136,9 @@ def __init__(self, log_outputs, colors, display_widget=True, display_handle=None
         self.display_widget = display_widget
         self.display_handle = display_handle
         self.logger_widget = logger_widget
+        if not self.display_widget or self.display_handle is None:
+            self.stream_handler = logging.StreamHandler()
```

Comment:
This is better- does the stream handler still need to be closed?

Reply:
I need to apply the formatting however, will let you know once done